docs/execution-providers/STVM-ExecutionProvider.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Standalone TVM (STVM)
+description: Instructions to execute ONNX Runtime with the Standalone TVM (STVM) execution provider
+parent: Execution Providers
+nav_order: 15
+redirect_from: /docs/reference/execution-providers/STVM-ExecutionProvider
```

Comment:
Don't think this redirect is needed since this is new, right?

Reply:
Ah I think we blindly applied the template from other EP markdown pages. Are these lines not necessary for new EP documentation pages?

Reply:
Not necessary - these were from old EP pages. It won't actually cause any harm but best to remove it

Reply:
I removed them to clean it up